Design-Automation_Version/DeleteWallsApp/DeleteWalls.cs

```diff
@@ -24,8 +24,8 @@ public ExternalDBApplicationResult OnShutdown(Autodesk.Revit.ApplicationServices
 
       public void HandleDesignAutomationReadyEvent(object sender, DesignAutomationReadyEventArgs e)
       {
-         e.Succeeded = true;
          DeleteAllWalls(e.DesignAutomationData);
+         e.Succeeded = true;
```

Comment:
if an error in thrown in `DeleteAllWalls()`, then `e.Succeeded` will not be called and test will be marked as failure.